package.json

```diff
@@ -39,6 +39,7 @@
     "test": "razzle test --env=jsdom",
     "start:prod": "NODE_ENV=production node build/server.js",
     "relay": "relay-compiler --src ./src --schema ./schema.graphql --persistFunction scripts/persistQuery.js",
+    "relay:clean": "rm -r src/__generated__",
```

Comment:
Might be overkill since you shouldn't have to do this more than once but thought it could make things easier :) 

I'll also update the README